hardware/raspberrypi/power/README.md

```diff
@@ -1,11 +1,39 @@
 # Power Supply
 
-The Raspberry Pi 3 is powered by a +5.1V micro USB supply. Exactly how much current (mA) the Raspberry Pi requires is dependent on what you connect to it. We have found that purchasing a 2.5A power supply from a reputable retailer will provide you with ample power to run your Raspberry Pi. You can purchase the [official Raspberry Pi Power Supply](https://www.raspberrypi.org/products/raspberry-pi-universal-power-supply/) from our website, and you can learn more about the differing power requirements of the various models of the Raspberry Pi on our [FAQ page](https://www.raspberrypi.org/help/faqs/#power). 
+The power supply requirements differ by Raspberry Pi model. All models require a 5.1V supply, but the current supplied generally increases according to model. All models up to the Raspberry Pi 3 require a microUSB power connector, whilst the Raspberry Pi 4 uses a USB-C connector.
 
-Typically, the model B uses between 700-1000mA depending on what peripherals are connected; the model A can use as little as 500mA with no peripherals attached. The maximum power the Raspberry Pi can use is 1 Amp. If you need to connect a USB device that will take the power requirements above 1 Amp, then you must connect it to an externally-powered USB hub.
+Exactly how much current (mA) the Raspberry Pi requires is dependent on what you connect to it. The following table gives minimum and typical current requirements. 
```

Comment:
"The following table gives minimum and typical current requirements. " - Hmmm, doesn't look like it actually does? :confused: 
"The following table gives minimal and recommended power requirements." :man_shrugging: